### PR TITLE
issue #84 - add grid-full option (see grid.html) and boxsizing border…

### PR DESCRIPTION
**Pull request by kotzendekrabbe**
…box which brokes nothing

### CODE REVIEW

File: docs/css/main.css [hunk 1]
```diff
@@ -44,6 +44,16 @@ ul, li {
   }
 }
 
+@media screen and (min-width: 998px){
+  body {
+    padding: 0;
+  }
+}
+
+*, *:after, *:before {
```
Comment (stefanjudis) [inline docs/css/main.css]: Can we go with `*::after`, `*::before*`? Single colon is only legacy support for IE8. :)

**Review by stefanjudis — CHANGES_REQUESTED** [review]
There are 12 reduntant `border-box` declaration then. Do we want to remove these maybe?

File: docs/css/main.css (hunk 1 above)
Comment (kotzendekrabbe) [reply docs/css/main.css]: you're right ;) also with reduntant declarations. thx

**Review by stefanjudis — APPROVED** [review]
Great stuff! 👍🏻🚢

**Review by zoepage — APPROVED** [review]

**Pull request merged**